src/Datadog.Trace/Tracer.cs

```diff
@@ -590,6 +591,9 @@ internal async Task WriteDiagnosticLog()
                     writer.WritePropertyName("agent_error");
                     writer.WriteValue(agentError ?? string.Empty);
 
+                    writer.WritePropertyName("appsec_enabled");
+                    writer.WriteValue(Security.Instance.Enabled);
+
                     writer.WriteEndObject();
```

Comment:
Should we include whether blocking is enabled here, and remove the other log message that duplicates those details?

Reply:
should be fixed in f6f9fe82